ember-toucan-core/src/components/form/file-input-field.ts

```diff
@@ -50,10 +50,6 @@ export default class ToucanFormFileInputFieldComponent extends Component<ToucanF
     return this.args.accept ?? '*';
   }
 
-  get multiple() {
-    return this.args.multiple ?? true;
```

Comment:
Just making sure: With this change, if they don't pass `@multiple`, it'll now end up `false`, whereas before, it would have defaulted to `true`.  That's what we want?

Reply:
That logic looks different, though...

`this.args.multiple ?? true;` means that it'll be `true` if they don't pass `@multiple`
`this.args.multiple ? true : undefined;` means that it'll be `undefined` if they don't pass `@multiple`

Reply:
Oh yeah this is coming back to me.

1. We want either `multiple` or  NO multiple attribute whatsoever, so we do want `undefined` NOT `multiple=false`
2. `Multiple=false` still weirdly allows users to select multiple files to upload which is NOT what we want.

Can verify that over here: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_multiple
1. try `multiple=false` 
2. try removing `multiple` altogether
3. Notice #1 and #2 do not give the same results. 

We validate that the `multiple` attribute is not there when `@multiple={{false}}` is passed in this test: https://github.com/CrowdStrike/ember-toucan-core/blob/main/test-app/tests/integration/components/file-input-field-test.gts#L331

So if all tests are passing in this PR, then we are getting the intended result (no `multiple` arg)!

Reply:
cool, that all makes sense.  So now, if the `@multiple` arg is not provided at all, the `multiple` HTML attribute will be omitted altogether, so the input will _not_ allow multiple selections.

Can we add `@multiple` to the documentation?

Reply:
Great idea! Let me add this...to the right doc